ui.apps/src/main/content/jcr_root/apps/sling/servlet/errorhandler/404.jsp

```diff
@@ -0,0 +1,2 @@
+<%@page session="false"%><%
```

Comment:
@rakeshkumar-verma : why are we including jsp pages here

Reply:
@rishabh-behl  this is default implementation as per acs.